src/index.js

```diff
@@ -12,12 +13,13 @@ async function initialize() {
 
     try {
         const initialPage = 1;
-        const episodes = await getEpisodes(initialPage);
+        const episodes = await getEpisodes(initialPage, _);
```

Comment:
You can make the filterName an optional argument on getEpisodes and did not use the _. This approach is more commun.